transformation_test/transformation_test.go

```diff
@@ -554,3 +569,14 @@ func sanitizeWriteLogEntriesRequest(t *testing.T, r *logpb.WriteLogEntriesReques
 	}
 	return req
 }
+
+func cleanStacktraceFromErrors(errors []any) []any {
+	for _, e := range errors {
+		errMap, ok := e.(map[string]any)
+		if !ok {
+			continue
+		}
+		delete(errMap, "stacktrace")
```

Comment:
[Optional] Can we not do this around line 491?

Reply:
This is actually something I didn't mean to leave in. It's from earlier experimenting with the tests, but it's not actually necessary.

Removed in [33a760b](https://github.com/GoogleCloudPlatform/ops-agent/pull/1722/commits/33a760be24be5089d762094ed6df3b609748e299)